overlord/ifacestate/handlers.go

```diff
@@ -197,7 +198,19 @@ func (m *InterfaceManager) setupProfilesForSnap(task *state.Task, _ *tomb.Tomb,
 		affectedSnaps = append(affectedSnaps, name)
 	}
 	sort.Strings(affectedSnaps)
-	return m.setupAffectedSnaps(task, snapName, affectedSnaps)
+	err = m.setupAffectedSnaps(task, snapName, affectedSnaps)
+
```

Comment:
maybe I'm confused but I don't see tests for these changes in handlers.go

Reply:
Test added